src/bufr/IodaEncoder/IodaDescription.cpp

```diff
@@ -96,16 +98,38 @@ namespace Ingester
                 variable.coordinates = nullptr;
             }
 
+            variable.range = nullptr;
             if (varConf.has(ConfKeys::Variable::Range))
             {
                 auto range = std::make_shared<Range>();
                 range->start = std::stoi(varConf.getStringVector(ConfKeys::Variable::Range)[0]);
                 range->end = std::stoi(varConf.getStringVector(ConfKeys::Variable::Range)[1]);
                 variable.range = range;
             }
-            else
+
+            variable.chunks = {1000};
+            if (varConf.has(ConfKeys::Variable::Chunks))
+            {
+                auto chunks = std::vector<ioda::Dimensions_t>();
+
+                for (const auto& chunkStr : varConf.getStringVector(ConfKeys::Variable::Chunks))
+                {
+                    chunks.push_back(std::stoi(chunkStr));
+                }
+
+                variable.chunks = chunks;
+            }
+
+            variable.compressionLevel = 6;
+            if (varConf.has(ConfKeys::Variable::CompressionLevel))
             {
-                variable.range = nullptr;
+                int compressionLevel = varConf.getInt(ConfKeys::Variable::CompressionLevel);
+                if (compressionLevel < 1 || compressionLevel > 6)
+                {
+                    throw eckit::BadParameter("GZip compression level must be a number 1-6");
```

Comment:
again double check this, I thought it was 1-9 for netCDF, is it different for IODA? And 0 I think means "off"

https://www.unidata.ucar.edu/software/netcdf/docs/group__variables.html#ga59dad3301f241a7eb86f31b339af2d26

Reply:
Also, is it Gzip or ZLib?

Reply:
It is Gzip (at least I'm setting the compressGZip property on ioda-engines for those variables). That doc is perfect (confirms 0 is a valid number). Thanks, I'm updating the code. Should be fixed shortly.